README.md

```diff
@@ -1,3 +1,36 @@
-# Backtrace-Javascript
+<p align="center">
+    <a href="https://backtrace.io/" target="_blank">
+        <img src="./sl.bt.png" alt="Backtrace" width="512">
+    </a>
+</p>
 
-Home of the all Backtrace's JavaScript integrations.
+# Sauce Labs backtrace-javscript SDKs
+
+These Javascript SDKs are a radical step forward in error reporting for web applications, with a number of exciting new features.
```

Comment:
Let's say it's one year from now. Is this description still valid?

Reply:
Nope. And overall i don't like it. Working on new copy.